crates/compiler/builtins/bitcode/src/list.zig

```diff
@@ -199,7 +199,7 @@ const Caller4 = fn (?[*]u8, ?[*]u8, ?[*]u8, ?[*]u8, ?[*]u8, ?[*]u8) callconv(.C)
 
 pub fn listMap(
     list: RocList,
-    caller: Caller1,
+    caller: Caller1, // zig 0.10 error: generic parameters not allowed in function with calling convention 'C'
```

Comment:
```zig
const Caller1 = fn (?[*]u8, ?[*]u8, ?[*]u8) callconv(.C) void;
```
I don't see a "generic parameter" here! Is this referring to the array size or something?

Reply:
This is due to the new function pointer syntax. If you update them all to:
```zig
const Caller1 = *const fn (?[*]u8, ?[*]u8, ?[*]u8) callconv(.c) void;
```
it should compile correctly. Unfortunately, the current compilation error is very confusing. Sorry for that.